.github/workflows/streaming-tests.yml

```diff
@@ -26,6 +26,10 @@ jobs:
         run: |
           pip install -e .
           pip install dandi
+      - name: Install HDMF-zarr # temporary
+        run: pip install hdmf-zarr
+      - name: Install latest HDMF # temporary
+        run: pip install -U hdmf
```

Comment:
It looks like it might be better to switch to a requirements-tests.txt file

Reply:
This is temporary b/c of hdmf-zarr's dependencies - it's normally not needed at all for that workflow

Reply:
Also the goal here is not to make such things directly be base requirements of nwbinspector (instead relying on `pynwb` to handle it); this is only temporary stuff for running the tests that will be ported to PyNWB after thorough evaluation using the NWB Inspector